tests/Browser/AdminViewTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Tests\Browser;
+
+use Laravel\Dusk\Browser;
+use Tests\Authentication;
+use Tests\DuskTestCase;
+
+class AdminViewTest extends DuskTestCase {
+
+    public function test_admin_calendar(): void {
+        $this->browse(function (Browser $browser) {
+            Authentication::login($browser);
+            $browser->visit('/dashboard/admin/calendar')
+                    ->assertSee('Calendar/News');
+            $browser->visit('/dashboard/admin/calendar/new')
+                    ->assertSee('New Calendar Event/News');
+        });
+    }
+    
+    public function test_admin_airports(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/dashboard/admin/airports')
+                    ->assertSee('Airport Management');
+        });
+    }
+
+    public function test_admin_scenery(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/dashboard/admin/scenery')
+                    ->assertSee('Scenery Management');
+            $browser->visit('/dashboard/admin/scenery/new')
+                    ->assertSee('New Scenery');
```

Comment:
I'd like to see these in their own tests as well. Because as we fill out these tests, we are going to want these to both be separate. I know it seems weird to do this right now, but I think it'll help when we start really building out tests

Reply:
Done.